ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -373,18 +377,91 @@ export default function TransactionList({
     nonceSortedCompletedTransactionsSelectorAllChains,
   );
 
-  const unfilteredCompletedTransactions = useMemo(() => {
-    return isTokenNetworkFilterEqualCurrentNetwork ||
-      overrideFilterForCurrentChain
+  console.log(
+    'unfilteredCompletedTransactionsAllChains',
+    unfilteredCompletedTransactionsAllChains,
+  );
```

Comment:
Double-check any logs before we merge.

Reply:
Addressed the logs in this file: `315d1770a7ac2d4cfe8203c438c57daa9ac09814`